### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Websites_Keyboard: Focus is not visible on some of the interactive controls on the page while navigating through the keyboard.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#103]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106e

**MAS ID Violated**: MAS2.4.7

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Sites" in the left menu section and Press Enter.
5. Verify that keyboard focus is clearly visible on all the interactive controls of the page while navigating through the keyboard.

**Actual Result:**
Focus is not visible on the following interactive controls of the page while navigating through the keyboard:
- "Create Website" button, "More" menu button, "Stop" and "Delete" menu items

**Expected Result:**
Whenever any interactive element receives keyboard focus, a rectangular border should appear around the control so that user can differentiate clearly between the focused and non-focused controls on the page.

**User Impact**
This Success Criterion helps anyone who relies on the keyboard to operate the page, by letting them visually determine the component on which keyboard operations will interact at any point in time.
People with attention limitations, short term memory limitations, or limitations in executive processes benefit by being able to discover where the focus is located.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

**Attachment:**
![bug 103_focus visibility issue](https://user-images.githubusercontent.com/38521133/44657280-ed7b8680-aa19-11e8-9366-5d1012d146d7.png)

![bug 103_focus visibility issue 2](https://user-images.githubusercontent.com/38521133/44657380-477c4c00-aa1a-11e8-92f8-cfd1ce7f6f41.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
